src/main/scala/Url.scala

```diff
@@ -1,3 +1,98 @@
-object Url {
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
+import akka.http.scaladsl.model.{ContentTypes, HttpEntity, StatusCodes}
+import akka.http.scaladsl.server.Directives._
+import akka.util.Timeout
+import cats.effect.IO
 
+import scala.concurrent.duration._
+import java.sql.Timestamp
+import scala.language.postfixOps
+import spray.json._
+import doobie._
+import doobie.implicits._
+import doobie.implicits.javasql._
+
+
+import scala.concurrent.ExecutionContext
+import scala.util.{Failure, Success}
+
+case class UrlIn(userId: Int, value: String, timestamp: String)
+case class UrlOut(userId: Int, value: String, timestamp: Timestamp)
+
+object DatabaseConnection {
```

Comment:
It would be good if you also check the comments from @amicuwehkamp from https://github.com/arcilli/browser-history/pull/14#pullrequestreview-733683027.

Reply:
Initially I didn't add that dependency in sbt, but we will put all the database logic into a single scala class and we'll go for the doobie-hikari